### DISCUSSION TITLE
Visual studio Code - namespace generating turn off

### DISCUSSION THREAD

**Discussion by Zakys98** [microsoft/vscode-cpptools#8314] [Ideas]
Hello,
I have got question about this.
https://stackoverflow.com/questions/69518605/visual-studio-code-namespace-generating-turn-off
Is there a way how to turn it off ?
If there is not, can you please add it as new feature ?
Thanks, for response.

**Comment by sean-mcmanus** [comment]
If you're using the C/C++ extension with a recent version of clang-tidy, then they should be turned off by default. It's controlled by the clang-format setting "FixNamespaceComments: false". It's possible you're using using another extension to format your code which doesn't have that default or your other clang-format settings are changing the default behavior.

**Reply by Zakys98** [reply]
Thank you, it works
